src/Core/StackExpression.php

```diff
@@ -92,4 +92,39 @@ public function addOptionOverride($key, $value)
 
         return $this;
     }
+
+    /**
+     * @return array
+     */
+    public function getVariablesOverrides()
+    {
+        return $this->overrides['variables'];
+    }
+
+    /**
+     * @param array $overrides
+     *
+     * @return self
+     */
+    public function setVariablesOverrides(array $overrides)
+    {
+        $this->overrides['variables'] = $overrides;
+
+        return $this;
+    }
+
+    /**
+     * Adds a single variable override to the variables overrides.
+     *
+     * @param string $key
+     * @param mixed  $value
```

Comment:
Please specify what "mixed" may be. Using piping might make it clearer i.e: string|int|null

Reply:
fixed